debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/SnapshotChangeRecordEmitter.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle;
+
+import io.debezium.data.Envelope.Operation;
+import io.debezium.pipeline.spi.OffsetContext;
+import io.debezium.relational.RelationalChangeRecordEmitter;
+import io.debezium.util.Clock;
+import oracle.streams.RowLCR;
+
+/**
+ * Emits change data based on a single {@link RowLCR} event.
+ *
+ * @author Gunnar Morling
+ */
+public class SnapshotChangeRecordEmitter extends RelationalChangeRecordEmitter {
```

Comment:
Is this class specific to oracle?

Reply:
I suppose that it's not. Let's move it to core once it's proven with SQL Server.